packages/tradershub/package.json

```diff
@@ -14,12 +14,13 @@
     },
     "dependencies": {
         "@deriv/api": "^1.0.0",
-        "@deriv/library": "^1.0.0",
         "@deriv/integration": "^1.0.0",
+        "@deriv/library": "^1.0.0",
         "@deriv/quill-design": "^1.3.2",
         "@deriv/quill-icons": "^1.0.10",
         "@deriv/react-joyride": "^2.6.2",
         "@deriv/utils": "^1.0.0",
+        "@headlessui/react": "^1.7.17",
```

Comment:
Do we need headless ui ? Aren't we using quill for UI components ? This does not make sense to me

Reply:
having tailwind, quill system and headlessui together in one package seems abit messy to me, I think it's better to prevent adding more ui libraries .

Reply:
@aizad-deriv why don't you update the quill design instead?